packages/core/src/parsers/jsx/element-parser.ts

```diff
@@ -190,6 +190,12 @@ export const jsxElementToJson = (
       if (types.isJSXAttribute(item)) {
         const key = item.name.name as string;
         const value = item.value;
+
+        // boolean attribute
+        if (!item.hasOwnProperty('value')) {
```

Comment:
I think we can reuse `value` from above
```suggestion
        if (value === null) {
```